docs/upgrade-smart-agent.md

```diff
@@ -0,0 +1,146 @@
+# Overview
+
+On XX/XX/XXX, SignalFx will launch the Smart Agent, version 5.0.
+
+At a high-level, to better integrate with Splunk, this new version of the Smart Agent will be signed with Splunk certificates.
+
+SignalFx recommends that you upgrade your Smart Agent as soon as possible.
+
+Current users can continue to run Smart Agent 4.X; however, after XX/XX/XXX, any new installs or upgrades will require you to change to the new Splunk-signed repositories.
+
+## Upgrade the Smart Agent
+
+Review the following options to learn how to upgrade the Smart Agent:
+
+### Option 1: ATP
+
+Note: For commands prefixed by #, you must run as root.
+
+1. Delete the existing SignalFx apt key:
+ - apt-key del 5AE495F6
```

Comment:
This comment applies to all commands throughout. Is it clear that these are commands to run in terminal? Before I had them formatted and prefixed with # (commonly used as a shell prefix of root commands) so it was implied. May need to make it more explicit or add additional formatting.

Reply:
Good point. I added some formatting to make the commands standout.